onnx_tf/handlers/frontend/space_to_batch_nd.py

```diff
@@ -0,0 +1,145 @@
+import numpy as np
+
+from onnx.helper import make_node
+from onnx.helper import make_tensor
+
+from onnx_tf.common.data_type import any_dtype_to_onnx_dtype
+from onnx_tf.common import get_unique_suffix
+from onnx_tf.handlers.frontend_handler import FrontendHandler
+from onnx_tf.handlers.handler import tf_op
+from onnx_tf.handlers.frontend.pad import Pad
+from onnx_tf.handlers.frontend.reshape import Reshape
+from onnx_tf.handlers.frontend.transpose import Transpose
+from onnx_tf.pb_wrapper import TensorflowNode
+
+
+@tf_op("SpaceToBatchND")
+class SpaceToBatchND(FrontendHandler):
+  SINCE_VERSION = 9
+
+  @classmethod
+  def version_9(cls, node, **kwargs):
+    pad_suffix = '_' + get_unique_suffix()
+    shape_suffix1 = '_' + get_unique_suffix()
+    shape_suffix2 = '_' + get_unique_suffix()
+    reshape_suffix1 = '_' + get_unique_suffix()
+    reshape_suffix2 = '_' + get_unique_suffix()
+    transpose_suffix = '_' + get_unique_suffix()
+
+    pad_output_name = node.outputs[0] + pad_suffix
+    shape_output_name1 = node.outputs[0] + shape_suffix1
+    shape_output_name2 = node.outputs[0] + shape_suffix2
+    reshape_output_name1 = node.outputs[0] + reshape_suffix1
+    transpose_output_name = node.outputs[0] + transpose_suffix
+
+    pad_name = 'pad' + pad_suffix
+    reshape_name1 = 'reshape' + reshape_suffix1
+    reshape_name2 = 'reshape' + reshape_suffix2
+    transpose_name = 'transpose' + transpose_suffix
+
+    # before pad, need to make sure the shape matches therefore
+    # paddings must be shape of (n, 2) instead of (m, 2)
+    block_shape = kwargs['consts']['SpaceToBatchND/block_shape']
+    input_paddings = kwargs['consts']['SpaceToBatchND/paddings']
+    input_shape = kwargs['consts']['SpaceToBatchND/input'].shape
```

Comment:
Hi @chinhuang007 , I'm fine with assuming block_shape and input_paddings as constant. But assuming `input` is a constant is too strong an assumption and will likely make this operator unusable in practice.

Also making these kinds of assumptions mean that you need to implement an "args_check". Otherwise the onnx-tf checker you implemented will not be able to check properly.